packages/core/src/table/table.scss

```diff
@@ -0,0 +1,205 @@
+// Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@import './../styles/tokens/generated/index';
+
```

Comment:
The list styles didn't make use of CSS custom properties. I believe that is largely because they were ported directly over from `clr/ui`. 

Would it make sense to have a few custom properites under `[cds-table]` to dry up all the repeated uses of global and alias design tokens? This would also make it easier for tables to be themed/customized independently of design tokens, reducing the cognitive load of having to understand how the aliases and global tokens work to change, say, the border radius of one table...

Reply:
I started with making everything an alias for the tables before using it, then reverted to just using globals. It would make sense to do some, but need to consider how many are meaningful. Any guidance there?

Reply:
@mathisscott Do you mean to localize properties like this? 

```
:host {
  --background: #{$cds-alias-object-container-background};
  --background-tint: #{$cds-alias-object-container-background-tint};
  --background-shade: #{$cds-alias-object-container-background-shade};
  --color: #{$cds-global-typography-color-400};
  --border-radius: #{$cds-alias-object-border-radius-100};
  --border-width: #{$cds-alias-object-border-width-100};
  --border-color: #{$cds-alias-object-border-color};
  --border-tint: #{$cds-alias-object-border-color-tint};
}```

Reply:
I think you could treat it like other web components localize just the vars to the table selectors 

```css
[cds-table] {
  --background: #{$cds-alias-object-container-background};
  --color: #{$cds-global-typography-color-400};
  --border-radius: #{$cds-alias-object-border-radius-100};
  --border: #{$cds-alias-object-border-width-100} solid #{$cds-alias-object-border-color};
}
```